Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -0,0 +1,275 @@
+        let graphicsOverlay = GraphicsOverlay()
+        
+        /// A line symbol to show the bounding extent.
+        private let boundingBoxSymbol = SimpleLineSymbol(style: .dash, color: .yellow, width: 3)
+        
+        /// A line symbol to show the attachment association.
+        private let attachmentSymbol = SimpleLineSymbol(style: .dot, color: .green, width: 3)
+        
+        /// A line symbol to show the connectivity association.
+        private let connectivitySymbol = SimpleLineSymbol(style: .dot, color: .red, width: 3)
+        
+        /// The feature layers that allow us to fetch the legend symbols of
+        /// different elements in the network.
+        private let featureLayers: [FeatureLayer] = [
+            .featureService.appendingPathComponent("1"), .featureService.appendingPathComponent("5")
+        ].map { FeatureLayer(featureTable: ServiceFeatureTable(url: $0)) }
+        
+        /// The utility network for this sample.
+        private var network: UtilityNetwork!
+        
+        /// The legends for elements in the utility network.
+        private(set) var legendItems: [LegendItem] = []
+        
+        // MARK: Methods
+        
+        /// Performs tasks including loading the network and adding legends.
+        func setup() async {
+            // Loads the utility network.
+            do {
+                // Updates the URL session challenge handler to use the
+                // specified credentials and tokens for any challenges.
+                ArcGISEnvironment.authenticationManager.arcGISAuthenticationChallengeHandler = ChallengeHandler()
+                
+                // Creates and loads the utility network.
+                network = UtilityNetwork(url: .featureService, map: map)
+                try await network.load()
+                await setStatusMessage("Tap on a container to see its content.")
+            } catch {
+                await setStatusMessage("An error occurred while loading the network.")
+            }
+        }
+        
+        deinit {
+            // Resets the URL session challenge handler to use default handling.
+            ArcGISEnvironment.authenticationManager.arcGISAuthenticationChallengeHandler = nil
+        }
+        
+        // MARK: Legend
+        
+        /// Creates legend items with a name and an image.
+        /// - Parameter displayScale: The display scale for the swatch images.
+        /// - Returns: An array of legends.
+        func updateLegendInfoItems(displayScale: CGFloat) async {
+            await setStatusMessage("Getting Legend Info…")
+            // The legend info array that contains all the info from each feature layer.
+            let legendInfos: [LegendInfo] = await withTaskGroup(of: [LegendInfo].self) { group in
+                var infos: [LegendInfo] = []
+                for layer in featureLayers {
+                    group.addTask { (try? await layer.legendInfos) ?? [] }
+                }
```

Comment:
It is good to keep variables as close to where they are used as possible.

```suggestion
                for layer in featureLayers {
                    group.addTask { (try? await layer.legendInfos) ?? [] }
                }
                var infos: [LegendInfo] = []
```